docs/docs/view-security.md

```
@@ -86,6 +86,18 @@ LAKEKEEPER__TRUSTED_ENGINES__TRINO__OWNER_PROPERTY=trino.run-as-owner
 LAKEKEEPER__TRUSTED_ENGINES__TRINO__IDENTITIES__OIDC__AUDIENCES=[trino]
 ```
 
+A request is matched to a trusted engine when **either** its token audience appears in `AUDIENCES` **or** its subject appears in `SUBJECTS`. Use `SUBJECTS` when the IdP does not mint tokens with a distinguishing audience — for example, to trust a specific service-account's subject UUID directly:
+
+```bash
+LAKEKEEPER__TRUSTED_ENGINES__TRINO__IDENTITIES__OIDC__SUBJECTS=["<trino-service-account-subject>"]
+```
+
+!!! note "When using the OPA bridge"
+
+    The [OPA bridge](./opa.md) authenticates to Lakekeeper with its own client credentials to evaluate permission checks. We recommend it runs under a **dedicated** Keycloak client with narrower privileges than the Trino catalog client — not a shared one.
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)